examples/blinky/main.c

```diff
@@ -1,14 +1,18 @@
+#include "libs/uart/api.h"
 #include <avr/io.h>
 #include <util/delay.h>
 
 #define LED0 (PD6)
 
 int main(void) {
     DDRD |= _BV(LED0);
```

Comment:
Can't wait til the GPIO library is a thing so I don't have to look at this anymore lol